src/connectors/connectStateResults.js

```diff
@@ -0,0 +1,45 @@
+const connectStateResults = (renderFn, unmountFn) => (widgetParams = {}) => ({
+  init({ instantSearchInstance }) {
+    renderFn(
+      {
+        state: undefined,
+        results: undefined,
+        instantSearchInstance,
+        widgetParams,
+      },
+      true
+    );
+  },
+
+  render({ results, instantSearchInstance, state }) {
+    const resultsCopy = Object.keys(results).reduce((acc, key) => {
```

Comment:
This could be simplified using destructuring:

```js
const { _state, ...resultsCopy } = results;
```

Reply:
Vue InstantSearch does not have babel set up for object.spread, because it uses bublé instead

Reply:
Yes, but here we're using the rest syntax, not spread.

Reply:
rest & spread both aren't supported

Reply:
I tested and it gets compiled to:

```js
var _state = results._state;
var rest = objectWithoutProperties( results, ["_state"] );
var resultsCopy = rest;
```

Seems correct to me :thinking:

Reply:
Another solution using `_objectSpread` helper:

```js
const resultsCopy = _objectSpread({}, results);
delete resultsCopy._state;
```